docs/reference/reference-intro.mdx

```diff
@@ -38,13 +38,13 @@ slug: /reference
   <div class="col col--6">
     <div class="card shadow">
       <div class="card__body">
-        <h3>OneSDK.js</h3>
+        <h3>OneSDK for Node.js</h3>
         <p>
-          Client library that communicates with registry and performs operations from profiles/maps.
+          Client library for working with use-cases in your application.
```

Comment:
"working with use cases" → I am sure it can me more exciting than this!

"Universal client library for for all the APIs you want to integrate with..."

Reply:
This is a reference and personally I would prefer precision over excitement here...